tests/common/database_fixture.hpp

```diff
@@ -246,11 +262,15 @@ struct database_fixture {
    const asset_object& create_bitasset(const string& name,
                                        account_id_type issuer = GRAPHENE_WITNESS_ACCOUNT,
                                        uint16_t market_fee_percent = 100 /*1%*/,
-                                       uint16_t flags = charge_market_fee);
+                                       uint16_t flags = charge_market_fee,
+                                       uint16_t precision = 2,
+                                       asset_id_type backing_asset = {});
```

Comment:
Why `{}`?

Reply:
I did this to get backing_asset to default to a default constructed asset_id_type. Wrong? Is there a better way?

Reply:
Parameter removed.